app/src/main/java/com/example/stretchy/activity/MainActivity.kt

```diff
@@ -88,50 +78,72 @@ class MainActivity : ComponentActivity() {
             BottomNavScreen(
                 stringResource(id = R.string.training),
                 painterResource(id = R.drawable.ic_training),
-                stringResource(id = R.string.training)
+                Screen.TrainingListScreen.route
             ),
         )
 
         Scaffold(
             bottomBar = {
-                BottomNavigation {
-                    val navBackStackEntry by navController.currentBackStackEntryAsState()
-                    val currentRoute = navBackStackEntry?.destination?.route
+                AnimatedVisibility(
+                    visible = showBottomNavBar,
+                    enter = slideInVertically(initialOffsetY = { it }),
+                    exit = shrinkVertically(),
```

Comment:
- [ ] activity has alot of code, check if there could be a functions to be added
- [ ] extract composables to other files